examples/text_summarization/abstractive_summarization_bertsumabs_cnndm.ipynb

```diff
@@ -0,0 +1,435 @@
+{
```

Comment:
<p>line 17 in this cell, is the unit actually "number of tokens"?</p><p>Should we not hard code NUM_GPUs? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/567/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>